bedrock/mozorg/templates/mozorg/home/home-voices.html

```diff
@@ -359,6 +359,14 @@ <h2 class="primary go">{{ _('Mozilla Developer Network') }}</h2>
   <button id="scroll-prompt">{{ _('Discover more') }}</button>
 </main>
 
+{% if l10n_has_tag('mozorg-home-fundraising-2016') and switch('mozorg-home-fundraising-2016') %}
```

Comment:
We still have a bunch of them :-\

At some point we should remove that template and clean up things.